src/corelibs/U2Formats/src/PDBFormat.h

```diff
@@ -104,7 +104,9 @@ class U2FORMATS_EXPORT PDBFormat : public TextDocumentFormatDeprecated {
 
         // Returns the end-of-name index for `specification`. Changes `readingMoleculeName` if needed.
         int returnEndOfNameIndexAndUpdateParserState(const QString& specification);
-
+        
+        // Check if IDs in list follows natural order
+        void checkIdsOrder(QList<int>& idSequence, U2OpStatus& ti);
```

Comment:
nit: 
1. Check -> Checks
2. Add details why `sequenceIds` is modified inside of the method.
Move the method into cpp file - there is not need to have it in the public interface: make it a static private cpp function